ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+v1.7.2 / 2018-02-20
+-------------------
+* Adding support for alert metrics like cpu-0.graph/cpu-idle.value
```

Comment:
Let's put the actual metric that this came up for rather than this made up example and also explicitly mention that this is about metrics that have multiple periods in their path.

Reply:
Addressed with 5070cc9